README.md

```diff
@@ -516,6 +516,21 @@ const clean = sanitizeHtml('<p><iframe src="https://us02web.zoom.us/embed/12345"
 });
 ```
 
+### Script Filters
+
+Similarly to iframes you can allow a script tag on a list of whitelisted domains or hostnames
+
+```js
+const clean = sanitizeHtml('<script src="https://www.safe.authorized.com/lib.js"></script>', {
```

Comment:
This example is confusing. The provided URL matches allowedScriptDomains but not allowedScriptHostnames. It would be better to use only one that makes sense in the example and separately document the other option.

Reply:
Thanks just changed this